src/transformState.js

```diff
@@ -4,8 +4,47 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
+function addProperties(object, objectModifier) {
+  for (const key in objectModifier) {
+    object[key] = objectModifier[key];
+  }
+
+  return object;
+};
+
+function removeProperties(object, objectModifier) {
+  for (const key of objectModifier) {
+    delete object[key];
+  }
+
+  return object;
+};
+
+function clearAll(object) {
+  for (const key in object) {
+    delete object[key];
+  }
+
+  return object;
+};
+
 function transformState(state, actions) {
-  // write code here
-}
+  for (const index of actions) {
+    if (index.type === 'addProperties') {
```

Comment:
Условные выражения можно переделать через switch/case, поскольку мы проверяем всего три варианта и у нас лишь одно условие - строгое сравнение (===), которое реализуется в switch/case по умолчанию.
```suggestion
    switch (index.type) {
      case 'addProperties':
      цикл из соответствующей функции
      break;
    } 
```

Reply:
@denys-anosov  Дякую, ти правий! Виправив)